index.html

```diff
@@ -1,16 +1,123 @@
+
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Cute Cakes</title>
 </head>
-
 <body>
-    <!-- Add your markup here -->
+    <main class="homepage-main">
+        <section class="homepage-hero">
+            <header>
+                <svg viewBox="0 0 100 80" width="40" height="40">
+                    <rect width="100" height="20"></rect>
+                    <rect y="30" width="100" height="20"></rect>
+                    <rect y="60" width="100" height="20"></rect>
+                </svg>
+                <img class="image" src="http://www.clker.com/cliparts/9/5/G/c/a/4/pink-frosted-cupcake-md.png" width="200px" alt="massive cake">
+                <p class="p-logo">Cute Cakes</p>
+                
+                <h1 class="h1header">The best cakes in town delivered to your door!</h1>
+                   <nav class="site-nav">
+                        <ul>
+                          <li><a href="#">Home</a></li>
+                          <li><a href="#">Cakes</a></li>
+                          <li><a href="#">Order</a></li>
+                          <li><a href="#">About</a></li>
+                          <li><a href="#">Contact</a></li>
+                        </ul>
+                    </nav>
+             </header>
+        </section>
+        
+        
+    <section class="homepage-feature"> 
+
+
+      <div>
+        <img src="https://media.istockphoto.com/photos/birthday-cake-decorated-with-colorful-sprinkles-and-ten-candles-picture-id1136810581?k=20&m=1136810581&s=612x612&w=0&h=Jj80oUK2CtY9nwYYRfJXB31ydjhOo5igRolQn-lV8M0=" alt="birthday celebtayion cake with sprinckles">
+    </div>
+            <div>
+                <h2 id="welcome">Welcome</h2>
+                <p>Hey! Glad to see you! Today is your day to taste the most delicious cakes and capcakes this side of heaven, look around, order and ENJOY!</p>
+                <button class="button">Order Now</button>
+            </div>
+            
+           
+    </section> 
+        
+        
+    <section class="homepage-products">
+           
+           <img src="img/istockphoto-1288470137-170667a.jpg" alt="pink cake image with sprincles">  
```

Comment:
The images were not properly positioned, so as mentioned during our calls, always wrap an image inside a container/div and set the image to 100% so that it follows the width of the container based on the grid system :)